cognite/client/config.py

```diff
@@ -36,7 +36,7 @@ class GlobalConfig:
 
     def __new__(cls) -> GlobalConfig:
         if hasattr(cls, "_instance"):
-            raise ValueError(
+            raise ValueError(  # TODO: raise a different error type?
```

Comment:
ChatGPT suggested `RuntimeError`, Python uses `TypeError` for `typing.Any`. I leave this up to you 😜